lms/djangoapps/course_api/views.py

```diff
@@ -158,11 +162,13 @@ class CourseListView(APIView):
             ]
     """
 
-    def get(self, request):
+    pagination_class = NamespacedPageNumberPagination
+    serializer_class = CourseSerializer
+    paginate_by = 20
```

Comment:
In that case, is there a reason why this view needs to override the default value of `settings.PAGINATE_BY`?